src/interfaces/action-arrival-payload.interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface ActionArrivalPayload {
+  id: string;
```

Comment:
It's used here https://github.com/madfish-solutions/templewallet-mobile/pull/181/files#diff-273b3530a2b438e6e3dd8cc2190bb7c2a853a2bc9d9c0dde1ac3ddfd05b8b4bcR13 for preventing other log entries to be opened after new actions arrive